pennylane/ops/__init__.py

```diff
@@ -39,13 +39,15 @@
 from .qubit import ops as _qubit__ops__
 from .qubit import obs as _qubit__obs__
 
+from .identity import GlobalPhase
```

Comment:
how is `Identity` imported? wondering why it isn't happening for GlobalPhase as well

Reply:
now imported in `ops/qubit/__init__` and included in the `__qubit_ops__` list like everything else